tests/e2e/README.md

```diff
@@ -0,0 +1,93 @@
+# End to end tests
+
+E2e tests largely follow the same syntax as [integration tests](tests/e2e/README.md). 
+Whereas integration tests are intended to mock and stress the back-end, server-side code, e2e tests the interface between front-end and back-end, as well as visual regressions with both assertions and visual comparisons.
+They can be run with make commands for the appropriate backends, namely:
+```shell
+make test-sqlite
+make test-pgsql
+make test-mysql
+make test-mysql8
+make test-mssql
+```
+
+Make sure to perform a clean front-end build before running tests:
+```
+make clean frontend
+```
+
+## Install playwright dependencies
+```
+npx install playwright --with-deps
+```
```

Comment:
I think we should automate this in `Makefile`, maybe with the evidence logic above.

Reply:
can you change to how it's now done with make

 @kdumontnu

Reply:
I changed it to `npx playwright install-deps`. We actually still need to install the system dependencies on most systems. I just couldn't figure out a way to automate that, and playwright will throw an error telling users to run the above command if deps are not installed, so I figured that's fine.